packages/flutter/lib/src/cupertino/text_selection_toolbar_button.dart

```diff
@@ -89,6 +85,10 @@ class CupertinoTextSelectionToolbarButton extends StatelessWidget {
   /// {@endtemplate}
   final String? text;
 
+  /// Whether the button is enabled or disabled. Buttons are disabled by default. To
+  /// enable a button, set its [onPressed] property to a non-null value.
```

Comment:
The first sentence should be it's own paragraph according to the styleguide.  So put an empty `///` line between the first sentence and the rest.

Reply:
Actually, it looks like this property isn't used outside of the `State` below.  Can you make it private?

Reply:
Actually, there's no need to make it private, I added this property when I tried removing `CupertinoButton` from `CupertinoTextSelectionToolbarButton` widget, but I ended up re-adding `CupertinoButton`, so this property is not needed anymore.